Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -8,7 +8,7 @@
     {
         <_DataGridPagination TItem="TItem" PaginationContext="@PaginationContext" PaginationTemplates="@PaginationTemplates" OnPaginationItemClick="@Paginate" New="@New" Edit="@Edit" Delete="@Delete" SelectedRow="@SelectedRow" ClearFilter="@ClearFilter" />
     }
-    <Table @ref="@tableRef" ElementId="@ElementId" Class="@ClassNames" Style="@Style" Margin="@Margin" Padding="@Padding" Striped="@Striped" Bordered="@Bordered" Borderless="@Borderless" Hoverable="@Hoverable" Narrow="@Narrow" Responsive="@Responsive">
+    <Table @ref="@tableRef" ElementId="@ElementId" Class="@ClassNames" Style="@Style" Margin="@Margin" Padding="@Padding" Striped="@Striped" Bordered="@Bordered" Borderless="@Borderless" Hoverable="@Hoverable" Narrow="@Narrow" Responsive="@Responsive" Resizable="@Resizable" ResizeMode="@(Enum.Parse<TableResizeMode>(ResizeMode.ToString("d")))">

```

Comment:
Just remove DataGridResizeMode and use ResizeMode directly.

Reply:
Fine, will update.

Reply:
Updated. Will be a simple fix breaking change. I will put it in the docs.

Reply:
Looking good. I just did small formating fixes in markdown.